tembo-operator/tests/integration_tests.rs

```diff
@@ -1837,14 +1900,13 @@ mod test {
 
     #[tokio::test]
     #[ignore]
-    async fn functional_test_ingress_route_tcp() {
-        // Initialize the Kubernetes client
+    async fn test_networking() {
         let client = kube_client().await;
+        let state = State::default();
 
-        // Configurations
         let mut rng = rand::thread_rng();
         let suffix = rng.gen_range(0..100000);
-        let name = &format!("test-ingress-route-tcp-{}", suffix.clone());
+        let name = &format!("example-dedicated-networking-{}", suffix.clone());
```

Comment:
Please change so this name variable is exactly 30 characters long after formatting.